### PR TITLE
Java get time in millis

### LINKED ISSUE #3097: [Term Entry] Java Calendar getTimeInMillis()
### Reason/inspiration (optional)

Please add the `getTimeInMillis()` method as a new term entry for the [Calendar](https://www.codecademy.com/resources/docs/java/calendar)�class. Provide the entry in a markdown file and place under the file path `content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md`

The entry should have the following:

�  An introductory paragraph about what the method is and does.
�  A syntax section with a pseudo code block with an explanation of it.
�  An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by bcaro1221**
### Description

This pull request is addressing the following [issue](https://github.com/Codecademy/docs/issues/3097). This is adding documentation for the Java method .getTimeInMillis() off of the Calendar class, which returns the calendar date and time represented in milliseconds. Closes #3097

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by bcaro1221** [comment]
@codecademy-docs is there a specific way I am to add reviewers? Also if the merge is blocked would I need to refetch the original repository?

**Comment by caupolicandiaz** [comment]
> @codecademy-docs is there a specific way I am to add reviewers? Also if the merge is blocked would I need to refetch the original repository?

The only work you are responsible for is updating the entry to address the review comments. All the other tasks are done by maintainers.

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+Title: '.getTimeInMillis()'
+Description: 'Returns the time in milliseconds.'
+Subjects: 
+	- 'Code Foundations'
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:5]: the indentations within the metadata section is incorrect. Please review one of our templates for reference.

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	- 'Computer Science'
+Tags:
+	- 'Date'
+	- 'Methods'
+CatalogContent:
+	- 'learn-java'
+	- 'paths/computer-science'
+---
+
+The **`.getTimeInMillis()`** method of the `Calendar` class returns the time represented by a calendar object in the form of milliseconds.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:15]:
```suggestion
The **`.getTimeInMillis()`** method of the `Calendar` class returns the time represented by a `Calendar` object in the form of milliseconds.
```

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Syntax
+
+```pseudo
+calendar.getTimeInMillis()
+```
+
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:22]: There should be some additional information here, such as what is the expected return value

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+## Example
+
+This example code creates a calendar object, records the current time, sets and records the time back to 12:00 AM the same day, and then prints the difference.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:25]:
```suggestion
This example code creates a `Calendar` object, records the current time, sets a new time, and then prints the difference.
```

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+```java
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:27]: the code should use 2 spaces for indentations not tabs

File: content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+		// Get the time in milliseconds of 12:00 AM today
+		long earlierToday = calendar.getTimeInMillis();
+
+		// Prints current amount of milliseconds that have passed today
+		System.out.println("Time passed today so far in milliseconds: " + (currentTime - earlierToday));
+	}
+}
+```
+
+The output of the code above is the time passed in milliseconds today from 12:00 AM.
```
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getTimeInMillis/getTimeInMillis.md:55]: The output should be in a ```shell block

And there should be a sentence before the block such as "The output of the code will be similar to:"

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@bcaro1221 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by bcaro1221** [comment]
Hi I'm having trouble with my stack of local commits at the moment and definitely would like some guidance before I push my newest commit with the suggested changes. So I tried to commit the changes and push them but then I got a message saying: 
```shell
error: failed to push some refs to 'https://github.com/bcaro1221/codecademy_docs.git'
hint: Updates were rejected because the remote contains work that you do not
hint: have locally. This is usually caused by another repository pushing to
hint: the same ref. If you want to integrate the remote changes, use
hint: 'git pull' before pushing again.
```

So I tried pulling and got the following message:

```shell
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
```

I tried looking this up and doing the recommended steps but the only thing that worked in regards to pulling was when I ran `git rebase`. But it looks like `git rebase` created a commit titled `Merge branch 'main' into JavaGetTimeInMillis` which looks like it merged master in to my newer commit. I was wondering if I should roll it back/if you all had any advice on what to do here (what the correct procedure is for fetching the origin updates after my pull request is already created?) or if this was the correct. @codecademy-docs @caupolicandiaz

**Review by caupolicandiaz — APPROVED** [review]
@bcaro1221 It can be difficult to diagnose the errors without more information. I went ahead and made the remaining edit directly. The branch is fine as is stands right now. This is ready now for a second review.

**Comment by bcaro1221** [comment]
Sgtm. Can I leave a feature request on the git lesson that asks to contribute to this open source resource? Specifically, in regards to this issue. Though I'm new to git I've had fair share of experience in the code review process, and I know that code review can sometimes take more than a day given people's schedules. Was wondering if y'all could add text specifically describing what to do after you get feedback on a pull request and there are new changes in the source/origin which need to be pulled in to your branch in order to make your changes? Thanks! @codecademy-docs @caupolicandiaz

**Comment by caupolicandiaz** [comment]
> Sgtm. Can I leave a feature request on the git lesson that asks to contribute to this open source resource? Specifically, in regards to this issue. Though I'm new to git I've had fair share of experience in the code review process, and I know that code review can sometimes take more than a day given people's schedules. Was wondering if y'all could add text specifically describing what to do after you get feedback on a pull request and there are new changes in the source/origin which need to be pulled in to your branch in order to make your changes? Thanks! @codecademy-docs @caupolicandiaz

I would recommend reviewing our contribution resources such as -
https://www.codecademy.com/resources/docs/contribution-guide/creating-a-pull-request-and-revisions

**Review by SSwiniarski — APPROVED** [review]
@bcaro1221 Looks good to me.

**Pull request merged**